### ISSUE TITLE
enable withoutExceptionHandling() in each test by default

### ISSUE THREAD

**Issue by gabbydgab** [laravel/ideas#2150]
## TL;DR
Is it possible to enable withoutExceptionHandling() in each test by default?

IMHO, it's inappropriate to commit a line with **$this->withoutExceptionHandling();** knowing that it will only halt your tests because you really want to validate if there are exceptions thrown that you didn't expect - in this example is a ViewException error.

We need to show the issues upfront during test execution. Rather than manually adding **$this->withoutExceptionHandling()** or  [debugging the response](https://laravel.com/docs/7.x/http-tests#debugging-responses) just to see what is going on.

I would like to know and discuss the PROS and CONS and to have a better understanding of this issue.

TIA

# My Problem
I'm trying to do TDD and I bump with this issue. It seems that my test will still PASS even if there's an exception error - particularly in this example, a ViewException error.

Here's what I have done. 
1. Create a test
```php
class SampleTest extends TestCase
{
     public function testTextMustBeSeen()
    {
           $this->get('/')->assertSeeText('show me this text')
    }
}
```
2. Controller
```php
class MyViewController extends Controller
{
     public function index(Request $request)
     {
           return view('sample');
     } 
}
```
3. View file (sample.blade.php)
```php
<h1>{{$title}}</h1>
```
4. Run the test and the result is **PASSED** even if I expect it to fail - as undefined variable $title.

# Initial Resolution
I've modified my test to something like this and my test is failing as expected.
```php
public function testTextMustBeSeen()
{
     $this->withoutExceptionHandling();
      $this->get('/')->assertSeeText('show me this text');
}
```
Once I debugged and fixed the issue, I removed the line **$this->withoutExceptionHandling();** then moved on.

As a workaround with this issue, I'm forced to add the following setUp code in the abstract TestCase class
```php
abstract class TestCase extends BaseTestCase
{
    use CreatesApplication;

    public function setUp(): void
    {
          parent::setUp(); 
          $this->withoutExceptionHandling();
    }
}
```

I would love to contribute to this but I don't know where and how to start.

**Comment by mfn** [comment]
But this has other implications, no?

Performing any HTTP request will immediately throw the exception / abort the test instead of giving you back a e.g. `Response` to e.g. check your API error responses and such?
